include/picolibrary/testing/unit/indicator.h

```diff
@@ -23,6 +23,7 @@
 #ifndef PICOLIBRARY_TESTING_UNIT_INDICATOR_H
 #define PICOLIBRARY_TESTING_UNIT_INDICATOR_H
 
+#include "gmock/gmock.h"
 #include "picolibrary/indicator.h"
 #include "picolibrary/testing/unit/random.h"
```

Comment:
The header file is currently relying on indirect includes. Please update to "include what you use".

```suggestion
#include "gmock/gmock.h"
#include "picolibrary/error.h"
#include "picolibrary/indicator.h"
#include "picolibrary/result.h"
#include "picolibrary/testing/unit/random.h"
#include "picolibrary/void.h"
```